src/os/FileJournal.cc

```diff
@@ -823,10 +828,10 @@ void FileJournal::write_header_sync()
   dout(20) << __func__ << " finish" << dendl;
 }
 
-int FileJournal::check_for_full(uint64_t seq, off64_t pos, off64_t size)
+int FileJournal::check_for_full(off64_t pos, off64_t size, bool skip_full)
 {
   // already full?
-  if (full_state != FULL_NOTFULL)
+  if (!skip_full && full_state != FULL_NOTFULL)
```

Comment:
![image](https://cloud.githubusercontent.com/assets/3147032/11745202/58889d94-a051-11e5-8ef5-0bc3061e153a.png)


we have reserve enough space for padding entry.